tests/linear_algebra/test_inv.py

```diff
@@ -0,0 +1,186 @@
+
+from unittest import TestCase
+
+import numpy as np
+from numpy.testing import assert_allclose
+from pytest import approx, mark
+
+from fastats.core.ast_transforms.convert_to_jit import convert_to_jit
+from fastats.linear_algebra import inv, matrix_minor, det
+
+
+inv_jit = convert_to_jit(inv)
+matrix_minor_jit = convert_to_jit(matrix_minor)
+det_jit = convert_to_jit(det)
+
+
+class MatrixInverseValidator:
+    """
+    This is a mixin class which tests both
+    the raw Python and the jit-compiled
+    version of the `inv()` function.
+    """
+    A, A_inv = None, None
+
+    def setUp(self):
+        self._A = np.array(self.A)
+
+    def test_inv_outputs_numpy(self):
+        A_inv = inv(self._A)
+        assert A_inv.tolist() == self.A_inv
+
+        I = np.eye(self._A.shape[0])
+        assert_allclose(self.A @ A_inv, I, atol=1e-10)
+
+    def test_inv_outputs_numba(self):
+        A_inv = inv_jit(self._A)
+        assert A_inv.tolist() == self.A_inv
+
+        I = np.eye(self._A.shape[0])
+        assert_allclose(self.A @ A_inv, I, atol=1e-10)
+
+    def test_det_numpy(self):
+        expected = np.linalg.det(self._A)
+        output = det(self._A)
+        assert expected == approx(output)
+
+    def test_det_numba(self):
+        expected = np.linalg.det(self._A)
+        output = det_jit(self._A)
+        assert expected == approx(output)
+
+
+class MathworldsInv2x2Test(MatrixInverseValidator, TestCase):
+    """
+    This test is an example 2x2 matrix inverse from
+    http://www.mathwords.com/i/inverse_of_a_matrix.htm
```

Comment:
there's a typo either in the class name or the docstring - mathworlds vs mathwords